src/components/movieDetails/index.jsx

```diff
@@ -9,27 +9,27 @@ function MovieDetails({ movie }) {
   }
 
   return (
-    <div className="movie-details">
-      <div className="movie-details-poster">
+    <div className={styles.details}>
+      <div className={styles.poster}>
         <img src={imageUrl} alt={name} />
       </div>
-      <div className="movie-details-info">
-        <div className="movie-details-header">
+      <div className={styles.info}>
+        <div className={styles.header}>
           <h3>{name}</h3>
-          <div className="movie-details-rating">{rating}</div>
+          <div className={styles.rating}>{rating}</div>
         </div>
-        <div className="movie-details-genres">
+        <div className={styles.genres}>
           {genres.map((genre, index) => (
-            <span key={index} className="movie-tile-genre">
+            <span key={index}>
               {genre}{index !== genres.length - 1 && ','}&nbsp;
             </span>
           ))}
         </div>
-        <p className="movie-details-timings">
+        <p className={styles.timings}>
           <span>{releaseYear}</span>
           <span>{convertMinutesToHours(duration)}</span>
```

Comment:
Kinda overkill in this case but still worth to mention is that u can memoize this kind of operations. Durations of movies can sometimes be equal and also u not sure about how they convert things in third party libs. Can be damaging for performance. 
So again, in this case all good but just remember these points doing something similar in more complicated circumstances.